packages/i18n-calypso-cli/README.md

```diff
@@ -14,12 +14,12 @@ Scans your JavaScript sources/build files and generates a POT file or a PHP tran
 ## From your code
 
 ```js
-var i18nCalypso = require( 'i18n-calypso-cli' );
+const i18nCalypso = require( 'i18n-calypso-cli' );
 i18nCalypso( {
-    inputPaths: <paths to your js files to scan>,
-    output: <path to your destination>
-    format: <format of the output: POT, PHP or JSON>
-    projectName: <Meta information about the project which can be used for autogenerated headers>
+	inputPaths, //<paths to your js files to scan>,
```

Comment:
I'd generally prefer a space between the `//` and the actual comment. I'd expect linting to catch that IMHO. 

Not a huge blocker though.

Reply:
Eslint to the rescue! We could enable https://eslint.org/docs/rules/spaced-comment

Reply:
I'm in favor, but it's work for another PR I'd expect. No need to flood this one.